description: >
    The original lane filter from Liam.
constructor: lane_filter.LaneFilterClassic
parameters:
  configuration:
      mean_d_0: 0
      mean_phi_0: 0
      sigma_d_0: 0.1
      sigma_phi_0: 0.1
      delta_d: 0.02
      delta_phi: 0.1

      # delta_d: 0.005
      # delta_phi: 0.025
      d_max: 0.3
      d_min: -0.15
      phi_min: -1.5
      phi_max: 1.5
      cov_v: 0.5
      linewidth_white: 0.0508 # 2 inches
      linewidth_yellow: 0.0254 # 1 inch
      lanewidth: 0.2413 # 2 ft (tile) - 2*linewidth_white - linewidth_yellow
      # lanewidth: 0.23 # old version 2016
      min_max: 0.1
      sigma_d_mask: 1.0
      sigma_phi_mask: 2.0
